core/src/main/java/com/linecorp/armeria/server/PathMapping.java

```diff
@@ -57,33 +57,27 @@
     String meterTag();
 
     /**
-     * Returns the exact path of this {@link PathMapping} if this is an exact path mapping,
-     * {@link Optional#empty()} otherwise.
+     * Returns the type of the path which was specified when this is created.
      */
-    Optional<String> exactPath();
+    RoutePathType pathType();
 
     /**
-     * Returns the prefix of this {@link PathMapping} if this is a {@link PrefixPathMapping} or a
-     * {@link PathMappingWithPrefix}, {@link Optional#empty()} otherwise.
+     * Returns the list of paths that this {@link Route} has. The paths are different according to the value
+     * of {@link #pathType()}. If the path type has a {@linkplain RoutePathType#hasTriePath() trie path},
+     * the first element is the path that represents the type and the second element is the trie path.
```

Comment:
this method will return a two-element list whose first element is ... and the second element is ...